shim/src/utilities.js

```diff
@@ -5,16 +5,18 @@
 
 export function throwTantrum(s, err = undefined) {
   const msg = `please report internal shim error: ${s}`;
+
   // note: we really do want to log these 'should never happen' things. there
   // might be a better way to convince the linter, though.
   // eslint-disable-next-line no-console
-  console.log(msg);
+  console.warn(msg);
```

Comment:
console.error, if they’re things that should never happen?

Reply:
Do we need a distinction form the actual error thrown?

Reply:
i'm just referring to the console message - "warn" is more for stuff that's worth fixing but not necessary to fix, "error" is for stuff that's an error - and people's console filtering settings might be set to "only error". not a big deal either way, ofc.